src/main/java/com/faforever/client/config/ClientProperties.java

```diff
@@ -150,6 +150,9 @@
     private String downloadUrlFormat;
     private String repoAndOwnerName;
     private String queryLatestVersionUrl;
+    private String queryVersionsUrl;
+    private int maxSupportedMajorVersion;
+    private int minSupportedMajorVersion;
```

Comment:
I would remove "minSupportedMajorVersion" (we have to have basic support all old versions forever to keep replays playable and even if we did we could build that in later) and all the corresponding checks.

Reply:
minSupportedMajorVersion is there as a just in case but never gets triggered since it is set to 0 it is there in case it ever would need to be used..